packages/authorization/src/types.ts

```diff
@@ -0,0 +1,246 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/authorization
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BindingAddress, InvocationContext} from '@loopback/context';
+
+/**
+ * Built-in roles
+ */
+export const EVERYONE = '$everyone';
+export const AUTHENTICATED = '$authenticated';
+export const UNAUTHENTICATED = '$unauthenticated';
+export const ANONYMOUS = '$anonymous';
```

Comment:
What's the difference between `UNAUTHENTICATED` and `ANONYMOUS`? In my mind, they are pretty much the same.

Reply:
does `ANONYMOUS` mean the user does not need authentication, and `UNAUTHENTICATED` mean the user didn't pass the authentication?
Can we also add a comment on them?

Reply:
cc @raymondfeng same question ^, could you explain the difference?